sbnobj/Common/POTAccounting/BNBSpillInfo.h

```diff
@@ -0,0 +1,63 @@
+#ifndef sbncode_BNBSpillInfo_HH
+#define sbncode_BNBSpillInfo_HH
```

Comment:
Take ~responsibility~ ~ownership~ the glory!
```suggestion
/**
 * @file sbnobj/Common/POTAccounting/BNBSpillInfo.h
 * @author Joseph Zennamo (jaz8600@fnal.gov)
 */

#ifndef sbncode_BNBSpillInfo_H
#define sbncode_BNBSpillInfo_H
```
(also, the file name is `.h`, so the include guard should go `_H`)

Reply:
switched